styleguide/source/_patterns/01-molecules/06-components/22-article-preview/02-article-preview-small/article-preview-small.twig

```diff
@@ -0,0 +1,7 @@
+<div class="topic-articles_article-preview">
```

Comment:
I'm unclear about the distinction between `topic-articles_article-preview` in the left rail and `topic_article-preview` in the hero. Should they both be singular?